src/nnops.jl

```diff
@@ -115,3 +118,20 @@ end
 function apply(normalization::UniformAdditiveNormalization, X::LazySet)
     return translate(X, fill(normalization.shift, dim(X)))
 end
+
+# solver using the CH of the sampled outputs as an inner approx of the real output
+@with_kw struct SampledApprox <: Solver
+    nsamples::Int = 10000
+end
+
+function NeuralVerification.forward_network(solver::SampledApprox, nnet, input)
+    samples = sample(input, solver.nsamples)
+    MIN = first(NV.compute_output(nnet, sample(input)))
```

Comment:
since we are using `Float64`, you could also initialize with `MIN = Inf` and `MAX = -Inf`